app/models/project/Project.scala

```diff
@@ -100,6 +100,20 @@ class ProjectDAO @Inject()(sqlClient: SQLClient)(implicit ec: ExecutionContext)
       parsed <- Fox.combined(r.toList.map(parse))
     } yield parsed
 
+  def findAllWithTaskType(taskTypeId: String)(implicit ctx: DBAccessContext): Fox[List[Project]] =
+    for{
+      accessQuery <- readAccessQuery
+      r <- run(
+        sql"""select distinct p._id, p._team, p._owner, p.name, p.priority, p.paused, p.expectedTime, p.isBlacklistedFromReport, p.created, p.isDeleted
+          from webknossos.projects_ p
+          join webknossos.tasks_ t on t._project = p._id 
+          join webknossos.taskTypes_ tt on t._taskType = tt._id
+          where tt._id = '#${taskTypeId}'"""
+          .as[ProjectsRow]
```

Comment:
```suggestion
        sql"""select #${columnsWithPrefix("a.")}
          from webknossos.projects_ p
          join webknossos.tasks_ t on t._project = p._id 
          join webknossos.taskTypes_ tt on t._taskType = tt._id
          where tt._id = '${taskTypeId}'"""
          .as[ProjectsRow]
```

Reply:
This is great, thanks! :)
I need the distinct, though. Otherwise one project shows up several times if it contains more than one task of the task type.

Reply:
oh, yes, you are right! I did not mean to remove the "distinct". good catch :)